test-app/spec/features/swagger_ui_spec.rb

```diff
@@ -5,8 +5,8 @@
   scenario 'browsing api-docs' do
     visit '/api-docs'
 
-    expect(page).to have_content('GET /blogs Searches blogs')
-    expect(page).to have_content('POST /blogs Creates a blog')
-    expect(page).to have_content('GET /blogs/{id} Retrieves a blog')
+    expect(page).to have_content('GET /blogs Searches blogs', normalize_ws: true)
+    expect(page).to have_content('POST /blogs Creates a blog', normalize_ws: true)
+    expect(page).to have_content('GET /blogs/{id} Retrieves a blog', normalize_ws: true)
```

Comment:
Capybara::Helpers::normalize_whitespace is deprecated, normalise_ws was added for backwards compatibility.
https://github.com/thoughtbot/capybara-webkit/issues/1065#issuecomment-411462204